RELEASING.md

```diff
@@ -0,0 +1,84 @@
+Releasing a new version
+=======================
+
+We will use the `v0.5.3` release (#738) as an example of how to release a new
+FDO version:
+
+* Fork the repo and create a new branch for the new release:
+
+    ```bash
+    gh repo fork fdo-rs/fido-device-onboard-rs --clone --remote
+    git pull upstream main
+    git checkout -b prepare-v0.5.3
+    ```
+
+* Update the `fido-device-onboard.spec` file and set the new version: `Version: 0.5.3`
+* Update the `version` within the `[[package]]` section in all the `Cargo.toml` files.
+* Update the `[[dependencies]]` in all the `Cargo.toml` files to use the latest
+FDO versions.
+* Update the `libfdo-data/fdo_data.h` file to reflect the correct version:
+
+    ```c
+    #define FDO_DATA_MAJOR 0
+    #define FDO_DATA_MINOR 5
+    #define FDO_DATA_PATCH 3
+    ```
+
+* Update `Cargo.lock` file:
+
+    ```bash
+    cargo update --offline --workspace
+    ```
+
+* Commit all the changes and create a PR (see #738 with all the changes described
+above):
+
+    ```bash
+    git add *
+    git commit -s -m "chore: bump for 0.5.3 release" -m "Prepare for the 0.5.3 release."
+    gh pr create
+    ```
+
+* Once all the tests passed, tag and sign the release:
+
+    ```bash
+    git tag -a -s v0.5.3
+    ```
+
+* Create the release in the github project by selecting the just created
+`v0.5.3` tag.
+
+Downstream releases
+-------------------
+
+* Fedora releases are handled by packit: A new bodhi update will be generated
```

Comment:
I think it's actually a new PR is generated in dist-git, not a new bodhi update.